identity/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.2.1 - unreleased
```

Comment:
This is missing a version bump in `identity/Cargo.toml`. Without the version bump this patch will not be released any time soon.

Reply:
Whoops. My bad for overlooking this.

Reply:
Sent follow-up in https://github.com/libp2p/rust-libp2p/pull/4125.